t/math/declare.xml

```diff
@@ -397,57 +397,57 @@
             <XMDual>
               <XMApp>
                 <XMTok meaning="list"/>
-                <XMRef idref="S2.Ex5.m1.3"/>
-                <XMRef idref="S2.Ex5.m1.4"/>
-                <XMRef idref="S2.Ex5.m1.5"/>
+                <XMRef idref="S2.Ex5.m1.9"/>
+                <XMRef idref="S2.Ex5.m1.10"/>
+                <XMRef idref="S2.Ex5.m1.11"/>
               </XMApp>
               <XMWrap>
-                <XMTok decl_id="S2.XMD9" font="italic" role="ID" xml:id="S2.Ex5.m1.3">g</XMTok>
+                <XMTok decl_id="S2.XMD9" font="italic" role="ID" xml:id="S2.Ex5.m1.9">g</XMTok>
                 <XMTok role="PUNCT">,</XMTok>
-                <XMDual role="FUNCTION" xml:id="S2.Ex5.m1.4">
+                <XMDual role="FUNCTION" xml:id="S2.Ex5.m1.10">
                   <XMApp>
                     <XMTok decl_id="S2.XMD10"/>
-                    <XMRef idref="XM30"/>
-                    <XMRef idref="XM31"/>
-                    <XMRef idref="XM32"/>
+                    <XMRef idref="S2.Ex5.m1.1"/>
+                    <XMRef idref="S2.Ex5.m1.2"/>
+                    <XMRef idref="S2.Ex5.m1.3"/>
```

Comment:
Really happy to see the `XM\d+` idrefs disappearing, I had seen them while working but hadn't gotten to asking what is up with them.

Reply:
Same here: I kept forgetting to track down where they came from. Basically a misnaming `getXMArgID` which really shouldn't be ID, but a stand-in for an id.  I'd change it if I came up with a good name!